sale_isolated_quotation/data/ir_sequence_data.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <!-- Sequences for sale.order -->
+        <record id="seq_sale_quotation" model="ir.sequence">
+            <field name="name">Quotation</field>
+            <field name="code">sale.quotation</field>
+            <field name="prefix">SQ</field>
+            <field name="padding">8</field>
```

Comment:
I would suggest you to use the current year to avoid to have so big padding:
`<field name="prefix">SQ/%(range_year)s/</field>`